src/includes/getting-started-primer/dart.mdx

```diff
@@ -1,5 +1,11 @@
 <Note>
 
-Pure Dart SDK used by any Dart application like AngularDart, CLI and Server, it enables reporting messages and errors.
+Sentry's Dart SDK enables automatic reporting of errors, messages, and exceptions. The SDK is available on GitHub [`sentry-dart`](https://github.com/getsentry/sentry-dart/). For Flutter apps, refer to [our Flutter documentation](/platforms/flutter) instead.
 
 </Note>
+
+Features:
```

Comment:
I believe we need to add a few more items here, even if generic since the SDK is able to do more than that.

For example:
Attachments, User feedback, and Performance monitoring.
See the Android feature list for inspiration https://sentry-docs-git-update-restructure-dart-docs.sentry.dev/platforms/android/

Reply:
I've done the best I can on this list of features based on what's listed for Android and what shows up as available for Dart, but you'll need to finalize this.